qt/scientific_interfaces/ISISReflectometry/test/RunsTable/RunsTablePresenterDisplayTest.h

```diff
@@ -245,6 +245,26 @@ class RunsTablePresenterDisplayTest : public CxxTest::TestSuite, RunsTablePresen
     verifyAndClearExpectations();
   }
 
+  void testNotifyBatchRowCellChanged() {
+    auto reductionJobs = twoGroupsWithMixedRowsModel();
+    auto presenter = makePresenter(m_view, reductionJobs);
+    auto const rowLocation = location(0, 0);
```

Comment:
Might be worth testing something other than index 0. Not sure it's worth a separate test but I tend to prefer to check the second group/row just because it's more likely to fail than finding the first one. It would also be clearer to create variables for the indices so it's clear how the expected outputs compare to the inputs (i.e. I'm not sure which `0` is which below, if that makes sense!)